public/index.html

```diff
@@ -2,10 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="https://res.cloudinary.com/fact0ry/image/upload/v1686770083/zero-assets/zos/favicon.png" />
+    <link rel="icon" href="https://framerusercontent.com/images/q863ucuBLgl5gWqHq00yE9u1TBc.png" />
```

Comment:
This should be uploaded to cloudinary first & then update the url here

Reply:
Ok, thanks!